editoast/src/views/work_schedules.rs

```diff
@@ -275,6 +295,238 @@ async fn project_path(
     Ok(Json(projections))
 }
 
+#[derive(Serialize, Deserialize, ToSchema)]
+struct WorkScheduleGroupCreateForm {
+    work_schedule_group_name: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, ToSchema)]
+struct WorkScheduleGroupCreateResponse {
+    work_schedule_group_id: i64,
+}
+
+#[utoipa::path(
+    post, path = "",
+    tag = "work_schedules",
+    request_body = WorkScheduleGroupCreateForm,
+    responses(
+        (status = 200, body = WorkScheduleGroupCreateResponse, description = "The id of the created work schedule group"),
+    )
+)]
+async fn create_group(
+    State(app_state): State<AppState>,
```

Comment:
```suggestion
    State(db_pool): State<DbConnectionPoolV2>,
```

I don't think you're using something else from the state, we have a shorthand for this case.

Reply:
The other handlers could be updated in the same way

Reply:
Fixed in a1bd117b3c855cecd04c1dee1aba55d9435fcdd9